src/api/app/views/webui/workflow_runs/index.html.haml

```diff
@@ -9,16 +9,11 @@
       .text-center
         %span.ml-3= page_entries_info(@workflow_runs)
       .accordion.pt-3#workflow-runs-accordion
-        .card
         - @workflow_runs.each do |workflow_run|
           .card-header{ id: "workflow-run-heading#{workflow_run.id}" }
             .mb-0
               %button.btn.btn-block{ type: 'button', data: { toggle: 'collapse', target: "#workflow-run#{workflow_run.id}" },
                                       aria: { expanded: 'false', controls: "workflow-run#{workflow_run.id}" } }
```

Comment:
```suggestion
```

Reply:
This shouldn't be needed anymore as well

Reply:
or you keep it and use it to link to the show view instead

Reply:
I've decided to drop it